app/controllers/accounts_controller.rb

```diff
@@ -1,9 +1,7 @@
 class AccountsController < ApplicationController
-  before_action :authenticate_user!
```

Comment:
Ah, sorry, must be late, I mis-read the file name.  Thanks for the clarification